packages/cfd/src/features/Components/passwordbox.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
some of the classnames are used by other components

changing the classname means updating other occurrences where this classname is used otherwise the UI will break

since they are in the cfd package, I moved them into `/features`